miqa/learning/nn_classifier.py

```diff
@@ -326,10 +342,10 @@ def forward(self, output, target):
         return loss
 
 
-def convert_bool_to_int(value):
-    if value == True:
+def convert_bool_to_int(value: bool):
+    if value:
         return 1
-    elif value == False:
+    elif not value:
```

Comment:
This entire function scares me... but it's worth mentioning that one of  `if value` and `if not value` will always be true, hence the below `else` block would never be entered. It seems like maybe this supports passing in types besides `bool`, but I have no idea how it's being used. For now, I'd suggest leaving the old behavior. If this was to fix a linting issue, maybe we should just use `is False` rather than `== False`.

Reply:
It was to fix a linting error. I changed it to use `is` instead of `==`.